socket.js

```diff
@@ -4,100 +4,99 @@
  * (c) 2014 Brian Ford http://briantford.com
  * License: MIT
  */
-
-angular.module('btford.socket-io', []).
-  provider('socketFactory', function () {
-
-    'use strict';
-
-    // when forwarding events, prefix the event name
-    var defaultPrefix = 'socket:',
-      ioSocket;
-
-    // expose to provider
-    this.$get = ['$rootScope', '$timeout', function ($rootScope, $timeout) {
-
-      var asyncAngularify = function (socket, callback) {
-        return callback ? function () {
-          var args = arguments;
-          $timeout(function () {
-            callback.apply(socket, args);
-          }, 0);
-        } : angular.noop;
-      };
-
-      return function socketFactory (options) {
-        options = options || {};
-        var socket = options.ioSocket || io.connect();
-        var prefix = options.prefix === undefined ? defaultPrefix : options.prefix ;
-        var defaultScope = options.scope || $rootScope;
-
-        var addListener = function (eventName, callback) {
-          socket.on(eventName, callback.__ng = asyncAngularify(socket, callback));
-        };
-
-        var addOnceListener = function (eventName, callback) {
-          socket.once(eventName, callback.__ng = asyncAngularify(socket, callback));
+(function(window, angular, undefined) {'use strict';
```

Comment:
Wrapping this is an unrelated change– if you're using CJS, this is especially redundant because your build tool already does it. Can you revert this change?